tests/tests/datm_cdeps_mx025_cfsr

```diff
@@ -10,15 +10,20 @@ export LIST_FILES="RESTART/MOM.res.nc \
                    RESTART/MOM.res_1.nc \
                    RESTART/MOM.res_2.nc \
                    RESTART/MOM.res_3.nc \
-                   RESTART/iced.2011-10-02-00000.nc \
-                   RESTART/DATM_CFSR.cpl.r.2011-10-02-00000.nc"
+                   RESTART/iced.2011-10-01-43200.nc \
+                   RESTART/DATM_CFSR.cpl.r.2011-10-01-43200.nc"
 
 export_datm_cdeps
-export WLCLK=40
 export DATM_SRC="CFSR"
 export FILENAME_BASE='cfsr.'
 export IATM=1760
 export JATM=880
+export DAYS="0.5"
+export FHMAX=12
+export RESTART_N=${FHMAX}
+export DT_DYNAM_MOM6='900'
+export DT_THERM_MOM6='1800'
+export CPL_SLOW=${DT_THERM_MOM6}
```

Comment:
Since you've reset the variable ``CPL_SLOW``, you also need to reset the variable ``export coupling_interval_slow_sec=${CPL_SLOW}`` in the test to have it take effect in ``nems.configure.datm.IN``, otherwise it the original (default=3600s) value will be used. 

I just tested this on Cheyenne; the nems.configure for this test shows:

```
runSeq::
@3600
   MED med_phases_prep_ocn_avg
   MED -> OCN :remapMethod=redist
   OCN
   @900
     MED med_phases_prep_ice
     MED -> ICE :remapMethod=redist
```

Reply:
@DeniseWorthen The line "export CPL_SLOW=${DT_THERM_MOM6}" will be removed from the following 4 tests:
datm_mx025_cfsr
datm_mx025_gefs
datm_cdeps_mx025_cfsr
datm_cdeps_mx025_gefs

In above 4 tests, 
coupling_interval_slow_sec=3600 seconds
coupling_interval_fast_sec=900 seconds
DT_DYNAM_MOM6=900 seconds (ocean dynamics time step)
DT_THERM_MOM6=1800 seconds (ocean thermodynamic time step)

After the changes, the 4 tests passed RT tests on wcoss_phase3. Thanks for discovering the issues.

Reply:
I don't believe that is the correct solution. If the slow coupling only occurs every hour, then MOM6 will take two internal thermodynamic timesteps before it gets "updated" forcing from the atm/ice.

Reply:
A better solution will be used in a future PR. An issue will be submitted. Thanks!

Reply:
OK, thanks. I think I mis-understood your original comment then.